webpack.config.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path');
```

Comment:
@ajainarayanan I though you say the normal lib build works fine -- do we really still need this?

Reply:
This is required for `font-awesome-sass-loader` Since we marked `font-awesome-*` as external in voyager we need to add this. This is to parse font awesome font files and serve them using `url-loader`.

Reply:
I removed the webpack build step to generate a library build explicitly for electron environment in the voyager PR. This however is required for using font-awesome loader in voyager.